microk8s-resources/actions/common/utils.sh

```diff
@@ -9,6 +9,16 @@ get_microk8s_group() {
   fi
 }
 
+get_microk8s_or_cis_group() {
+  if [ -e $SNAP_DATA/var/lock/cis-hardening ]
+  then
+    echo "root"
+  else
+    get_microk8s_group
+  fi
+}
+
+
```

Comment:
I would much rather we kept this as `get_microk8s_group`. Reading `get_microk8s_or_cis_group` in a foreign place might be misleading.

Would we ever need the actual group if we have applied CIS hardening?

Reply:
> Would we ever need the actual group if we have applied CIS hardening?

If we have applied CIS hardening the group of the files we creates should be root and not microk8s/snap_microk8s. This means that whenever we create a file we would either need to check if we are in CIS or push this check into a function like `get_microk8s_or_cis_group`. In practice this is not in many places, but I thought it would be better if we keep this check in a single place.

Reply:
Should it not be just `get_microk8s_group` then that takes CIS into account?

Reply:
Ah, we have kept the `get_microk8s_group` as it was. It is still used in some places.